fetch_archives.py

```diff
@@ -0,0 +1,30 @@
+import pyvo
+
+
+def fetch_archives():
+    archive_list = pyvo.registry.search(servicetype="tap")
+
+    with open('tools/archives/pyvo_integration/tool-data/astronomical_archives_gen.loc', 'w') as f, open('tools/archives/pyvo_integration/tool-data/astronomical_archives_gen.loc.sample', 'w') as f1:
```

Comment:
Could you remind me why do we need both files? They look completely the same

Reply:
If I remember correctly you need to have a sample file otherwise it will generate some error when trying to publish the tool (we had that problem when we tried the first time)

The first version of the sample file only had one line of data but then we just used the same content for simplicity sake I guess. Will try to find the pull request where we talked about that and post it there

Reply:
Ok, the sample is needed when using "data managers", that are used to update *.loc files (normally, these files are used to keep info about "reference data"). As soon as there is no such "reference datasets" used by the tool, we stick to direct update. 
Sample file can be empty, only having column description. But if it works like this, let's not touch it:)